zebra-chain/src/transaction/serialize.rs

```diff
@@ -266,21 +308,28 @@ impl ZcashDeserialize for Transaction {
                 let outputs = Vec::zcash_deserialize(&mut reader)?;
                 let lock_time = LockTime::zcash_deserialize(&mut reader)?;
                 let expiry_height = block::Height(reader.read_u32::<LittleEndian>()?);
+
                 let value_balance = (&mut reader).zcash_deserialize_into()?;
+                let anchor = ();
                 let mut shielded_spends = Vec::zcash_deserialize(&mut reader)?;
                 let mut shielded_outputs = Vec::zcash_deserialize(&mut reader)?;
+
                 let joinsplit_data = OptV4Jsd::zcash_deserialize(&mut reader)?;
 
                 use futures::future::Either::*;
-                let shielded_data = if !shielded_spends.is_empty() {
-                    Some(ShieldedData {
+                let sapling_shielded_data = if !shielded_spends.is_empty() {
+                    Some(sapling::ShieldedData {
+                        value_balance,
+                        anchor,
```

Comment:
There's no shared anchor here, so using a variable is a bit confusing:
```suggestion
                        anchor: (),
```

Reply:
I opened https://github.com/ZcashFoundation/zebra/pull/1955 for this change to the RFC